OracleWebLogic/dockerfiles/12.2.1.4/Dockerfile.generic

```diff
@@ -94,6 +94,7 @@ COPY --from=builder --chown=oracle:oracle /u01 /u01
 COPY container-scripts/createAndStartEmptyDomain.sh container-scripts/create-wls-domain.py /u01/oracle/
 
 RUN chmod +xr $SCRIPT_FILE && \
+    chown oracle:oracle -R /u01 && \
     chown oracle:oracle -R /u01/oracle/createAndStartEmptyDomain.sh && \
```

Comment:
`chown -R` means apply recursively, so if you run at /u01, then you no longer need the following lines specifically for the scripts.  Could we do this at /u01/oracle?

Reply:
the problem is that the Oracle Home is at /u01 and the problem with the introspection job is due to permissions to the /u01 directory.  Do you mean replace the two scripts chown statements with one "chown oracle:oracle -R /u01/oracle" or to remove all together the chown statement for the scripts?

Reply:
I was asking two different things: a) If `chown -R /u01/oracle` was sufficient?, and b) either way, once we have `chown -R /u01` or `chown -R /u01/oracle` then the other lines in the same statement are unnecessary since those paths are under /u01/oracle.